location/models.py

```diff
@@ -591,10 +594,8 @@ def get_user_districts(cls, user):
         cachedata = cache.get(f"user_districts_{user.id}")
         districts = []
         if cachedata is None:
+            cache_location_graph_if_empty()
             cache_district = cache.get("location_types")
```

Comment:
Keep naming consistent with the content. Should be something like locations_by_type.

Reply:
No it does both type and graph